dev/tools/gen_date_localizations.dart

```diff
@@ -157,13 +157,18 @@ Iterable<String> _materialLocales() sync* {
 }
 
 Map<String, File> _listIntlData(Directory directory) {
-  final Map<String, File> result = <String, File>{};
+  final Map<String, File> localeFiles = <String, File>{};
   for (FileSystemEntity entity in directory.listSync()) {
     final String filePath = entity.path;
     if (FileSystemEntity.isFileSync(filePath) && filePath.endsWith('.json')) {
       final String locale = path.basenameWithoutExtension(filePath);
-      result[locale] = entity;
+      localeFiles[locale] = entity;
     }
   }
-  return result;
+
+  final List<String> locales = localeFiles.keys.toList(growable: false);
```

Comment:
Doesn't matter too much but `SplayTreeMap` might be more concise.

Reply:
I always forget that we have one in `dart:collection` because I use it so rarely.